setup.cfg

```diff
@@ -21,9 +21,6 @@ classifiers =
     Programming Language :: Python
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
```

Comment:
Normally I don’t like these being removed, but since this is not user facing and doesn’t have much Python, I think it’s okay.

Reply:
It’s now an optional flag. See https://github.com/scikit-hep/boost-histogram/blob/5c7ae1ca9a849ebf53cad5b7441a0d33e6e66b8d/.pre-commit-config.yaml#L35

I’d usually add that, but here I think it’s okay to leave it off due to reasons above.

Reply:
Ah, thanks for letting me know about the option. Indeed agreed that here is fine :-).